src/tasks/batch_integration/api/file_solution.yaml

```diff
@@ -0,0 +1,61 @@
+type: file
+example: "resources_test/batch_integration/pancreas/unintegrated.h5ad"
+info:
+  label: "Solution"
+  summary: Solution dataset
+  slots:
+    layers:
+      - type: integer
+        name: counts
+        description: Raw counts
+        required: true
+      - type: double
+        name: normalized
+        description: Normalized expression values
+        required: true
+    obs:
+      - type: string
+        name: batch
+        description: Batch information
+        required: true
+      - type: string
+        name: label
+        description: label information
+        required: true
+    var:
+      - type: boolean
+        name: hvg
+        description: Whether or not the feature is considered to be a 'highly variable gene'
+        required: true
+    obsm:
+      - type: double
+        name: X_pca
+        description: The resulting PCA embedding.
+        required: true
+    obsp:
+      - type: double
+        name: knn_distances
+        description: K nearest neighbors distance matrix.
+        required: true
+      - type: double
+        name: knn_connectivities
+        description: K nearest neighbors connectivities matrix.
+        required: true
+    uns:
+      - type: string
+        name: dataset_id
+        description: "A unique identifier for the dataset"
+        required: true
+      - type: string
+        name: normalization_id
+        description: "Which normalization was used"
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: object
+        name: knn
```

Comment:
Is this the same information as in `.uns['neighbors']` that `scanpy.neighbors` generates?

Reply:
Yepyep! Should I rename it to `knn_neighbors`?

Reply:
'neighbors' is the default and what most methods expect out of the box, so I'd go with that